pom.xml

```diff
@@ -80,7 +80,7 @@
 				<liquibasePluginArtifactId>liquibase-maven-plugin</liquibasePluginArtifactId>
 				<liquibasePluginVersion>2.0.1</liquibasePluginVersion>
 				<liquibaseDemoDataFileName>liquibase-demo-data.xml</liquibaseDemoDataFileName>
-				<liquibaseMVPDataFileName>liquibase-mvp-data.xml</liquibaseMVPDataFileName>
+				<liquibaseCIELDataFileName>liquibase-CIEL-data.xml</liquibaseCIELDataFileName>
```

Comment:
I'd probably keep CIEL in lower case here, even though this means renaming the file.